package.json

```diff
@@ -229,9 +229,8 @@
     "watch": "tsc -watch -p ./ && node scripts/copyVendor.js",
     "postinstall": "node ./node_modules/vscode/bin/install",
     "test": "npm run compile && node ./node_modules/vscode/bin/test",
-    "check": "gts check",
-    "clean": "gts clean",
-    "fix": "gts fix",
+    "check": "./node_modules/eslint/bin/eslint.js . --ext .js,.jsx,.ts,.tsx",
```

Comment:
Does eslint provide command as something like eslint instead of directly call bin/eslint?

Reply:
Use eslint from node_modules path allow us to use it without install eslint globally.
Outside user trigger `npm run check` to run eslint check and `npm run fix` to run eslint auto fix.